MIP104/MIP104.md

```diff
@@ -426,14 +254,6 @@ List of current active Arrangers:
 
 ¤¤¤
 
-#### 5.1.2: Arranger Standard Fees
-
-Temporarily, no arranger maximum fees are enforced.
-
-#### 5.1.3: Arranger Concentration Risk Management
-
-The Stability Facilitators must take action to search and propose onboarding of additional Arrangers when it is economically feasible to do so, in order to help reduce the concentration risk to individual Arrangers. Strict limits to the amount of assets that each Arranger services for must be developed and implemented in this section over time.
-
 #### 5.1.4: Arranger Reports and Stress Tests
 
 Arrangers must publish monthly reporting on each Arranged Structure they have arranged, and must every six months also publish stress test analysis that shows how the structures would perform based on historical financial crisis scenarios and other example scenarios. The Stability Facilitators must periodically fund independent Ecosystem Actors to review and verify the quality and the results of the stress tests. Should an independent review produce an unfavorable result, the Responsible Facilitators must propose a governance poll for warning, temporarily deactivating, or permanently offboarding the Arranger and/or the Asset Managers connected to the discovered issue.
```

Comment:
There is a typo in point 5.1.4.1 below, in the second bullet (I don't know why I cannot insert the comment there) - the last word should be "expenses", not "expensesi".